.github/workflows/release.yml

```diff
@@ -7,29 +7,7 @@ on:
 
 jobs:
   release:
-    name: Release
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@master
-        with:
-          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
-          fetch-depth: 0
-
-      - name: Setup Node.js 16.x
-        uses: actions/setup-node@master
-        with:
-          node-version: 16.x
-
-      - name: Install Dependencies
-        run: npm ci
-
-      - name: Create Release Pull Request or Publish to npm
-        id: changesets
-        uses: changesets/action@master
-        with:
-          # This expects you to have a script called release which does a build for your packages and calls changeset publish
-          publish: npm run release
-        env:
-          GITHUB_TOKEN: ${{ secrets.CHANGESETS_PAT }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    uses: hashicorp/web-platform-packages/.github/workflows/release.yml
```

Comment:
[question] will this always pull from the `main` branch?

Reply:
> [question] will this always pull from the `main` branch?

Yes, the previous/ existing workflow defines `main` as the branch to pull from.